lib/streamlit/delta_generator.py

```diff
@@ -353,6 +354,28 @@ def _enqueue(
 
         return _value_or_dg(return_value, output_dg)
 
+    def container(self):
+        return self._block()
+
+    # TODO: Enforce that columns are not nested or in Sidebar
+    def columns(self, weights):
+        if isinstance(weights, int):
+            # If the user provided a single number, expand into equal weights.
+            # E.g. 3 => (1, 1, 1)
+            # TODO: check that the number > 0
+            weights = (1,) * weights
+
+        def column_proto(weight):
+            col_proto = Block_pb2.Block()
+            col_proto.column.weight = weight
+            return col_proto
+
+        horiz_proto = Block_pb2.Block()
+        horiz_proto.horizontal.unused = False
```

Comment:
Yeah, needed to set something so that I could check against the presence of the Horizontal() proto later.

Set it to horizontal.unused = True, which I think makes a bit more sense.